app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/ui/SearchScreen.kt

```diff
@@ -218,18 +262,18 @@ private fun RecipeTitle(title: String) {
 }
 
 @Composable
-private fun RecipeImage(imageUrl: String) {
+private fun RecipeImage(modifier: Modifier = Modifier, imageUrl: String) {
```

Comment:
In cases like this (a param without default value, and a param with default value), the param with default value should be the second (or third - after all params without default value). That way a function can be called just by passing imageUrl, without having to use named args.

Reply:
Ok. Android studio says that the modifier should be first, but probably when passing it ...

Reply:
Modifier should always be the first param with default value.
This is by convention only though - for consistency.
 i.e.:
```Kotlin
@Composable
fun someComposable(
      param1: SomeType,
      param2: String,
      param3: Long,
      modifier: Modifier = Modifier, // modifier is the first param among ones specifying the default value
      param4: Int = 1,
      param5: Dp = 5.dp,
      ...
) {
    // ...
}
```